### PR TITLE
add debug logging

### PR DESCRIPTION
**Pull request by pfrazee**
This is part of an overall effort to add debug output to all of the dat network stack.

To test, run `DEBUG=discovery-channel npm test`.

Double-check my output in case it's wrong or incomplete.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -188,3 +199,8 @@ function noHash (id) {
   if (typeof id === 'string') return Buffer(id)
   return id
 }
+
+function shortHex (buf) {
+  buf = buf.toString('hex')
+  return buf.slice(0, 6) + '..' + buf.slice(-2)
+}
```
Comment (mafintosh) [inline index.js]: you should put this function into a module (pretty-hash)

**Comment by mafintosh** [comment]
bikeshed comment: i like the `debug('foo=%s', 'bar')` syntax more but no big deal

**Comment by pfrazee** [comment]
oh wow TIL, I didn't know console.log supported %s. I shouldve done that

**Comment by pfrazee** [comment]
Ok, I'll make pretty-hash and update all the active PRs to use it

**Comment by pfrazee** [comment]
Updated

**Comment by pfrazee** [comment]
bump

**Comment by mafintosh** [comment]
5.4.0

**Pull request merged**